docs/setup-xdebug.md

```diff
@@ -0,0 +1,16 @@
+## Development
+
+### OCC
+
+Run inside of the Nextcloud container:
+
+```bash
+set XDEBUG_CONFIG=idekey=PHPSTORM
+sudo -E -u www-data php -dxdebug.remote_host=192.168.21.1 occ
+```
+
+### Useful commands
+
+- Restart apache to reload php configuration without a full container restart: `docker compose kill -s USR1 nextcloud`
+- Access to mysql console: `mysql -h $(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_database-mysql_1) -P 3306 -u nextcloud -pnextcloud`
+- Run an LDAP search: `ldapsearch -x -H ldap://$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_ldap_1) -D "cn=admin,dc=planetexpress,dc=com" -w admin -b "dc=planetexpress,dc=com" -s subtree <filter> <attrs>`
```

Comment:
These commands are not related to xdebug

Reply:
Yes, I didn't understand if it's related to it... So, I fixed it in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/998b7a06edd4dc4552bf0402f09cfff1eb26a33f

You can find the documentation on the "useful commands" here : https://github.com/arawa/nextcloud-docker-dev/blob/doc/improve-getting-started/docs/useful-commands.md

I added a link to this file here : https://github.com/arawa/nextcloud-docker-dev/blob/1422c78fe8e146f92a406261a1527b0d78b6bdf7/README.md?plain=1#L82